srcpkgs/msitools/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'msitools'
+pkgname=msitools
+version=0.101
+revision=1
+short_desc="A set of programs to inspect and build Windows Installer (.MSI) files"
+maintainer="Alex Franchuk <alex.franchuk@gmail.com>"
+license="LGPL-2.1-or-later"
```

Comment:
The copyright file suggests it's not this simple https://gitlab.gnome.org/GNOME/msitools/-/blob/master/copyright

Reply:
I've updated it. The `DISCLAIMED` copyright is omitted since it is explicitly releasing any copyright.